src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/mapper/HaFlowMapper.java

```diff
@@ -73,4 +81,52 @@ public abstract class HaFlowMapper {
     @Mapping(target = "trackLldpConnectedDevices", ignore = true)
     @Mapping(target = "trackArpConnectedDevices", ignore = true)
     public abstract FlowEndpoint toSubFlowSharedEndpointEncapsulation(HaSubFlow haSubFlow);
+
+    /**
+     * Convert {@link SwitchId} to {@link Switch} object.
+     */
+    public Switch newSwitch(SwitchId switchId) {
```

Comment:
It is a bit strange to see switch converter method in the HaFlowMapper. I also don't see usages in this PR. Why is it public here?

Reply:
I see, thanks for the explanation. Still maybe move it to the switch converter and use it here via `uses` ?

Reply:
We could, but there is no SwitchConverter in this module, So i'll have to create it for only one conversion. Honestly conversion of `switchId` to `switch` look s a little bit strange. But I have to use it to keep `Switch` object in HaFlow. 
So I don't really want to move it because this approach is kind of workaround.